packages/ui-components/src/Avatar/Avatar.module.scss

```diff
@@ -0,0 +1,98 @@
+@import "../styles/tokens.scss";
+
+@mixin selectable-border($n) {
+  box-sizing: border-box;
+  border: 2px solid $n;
+}
+
+.avatar {
+  position: relative;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  flex-shrink: 0;
+  font-family: Source Sans Pro, sans-serif; // TODO (koorosh): replace with fonts from tokes
+  font-style: normal;
+  font-weight: 600;
+  line-height: 1;
+  border-radius: 50%;
+  overflow: hidden;
+  user-select: none;
+}
+
+.intent-default {
+  background: $crl-neutral-2;
+  color: $crl-neutral-7;
+}
+
+.intent-info {
+  background: $crl-blue-1;
+  color: $crl-blue-3;
```

Comment:
Wherever possible (or unless there's a specific reason not to do so) let's refer to the `base` aliases, in this case `$crl-base-blue`.  The reason being the `base` aliases will be the most common colors seen in the codebase and, conversely, the explicit numerical values should be uncommon enough to call themselves out as a idiosyncratic use case when they do show up.

Reply:
Gotcha!